roles/pulp/defaults/main.yaml

```diff
@@ -14,5 +14,7 @@ pulp_content_container_name: pulp-content
 pulp_worker_container_name: pulp-worker
 
 pulp_content_origin: "http://{{ ansible_fqdn }}:24816"
+pulp_rhsm_url: "https://{{ ansible_fqdn }}/rhsm"
```

Comment:
I was debating this: if we don't expose the setting (or it has no value), how does Katello interpret this? It's defined [here](https://github.com/Katello/katello/blob/83498c4f1e027faf730f6d40d27dcdf309d0af1e/app/models/katello/concerns/smart_proxy_extensions.rb#L605-L615).

What if we enhance this that if it isn't set, Katello will show its own RHSM API URL? That way you can present a Pulp server without also the RHSM API. Because essentially this is really the `{{ foreman_url }}/rhsm` value unless it's a content proxy (which we don't support yet).

Reply:
https://github.com/Katello/katello/pull/11328 takes a stab at that.